### PR TITLE
Migrated from TravisCI to Github Actions

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Migrated from TravisCI to Github Actions
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | CI is :green_circle: https://github.com/Progi1984/prestonbot/runs/2038588782 (7.3 is red because can't push to @coveralls)
| Possible impacts? | N/A

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+name: PHP
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.1', '7.2']
```
Comment (atomiix) [inline .github/workflows/php.yml:10]: Prestonbot runs on php 7.3 ;)
```suggestion
        php-versions: ['7.1', '7.2', '7.3']
```

File: .github/workflows/php.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    - name: Install dependencies
+      run: |
+        composer install --no-interaction
+        ./vendor/bin/simple-phpunit install
+
+    - name: PHPUnit
+      run: SYMFONY_DEPRECATIONS_HELPER=weak ./vendor/bin/simple-phpunit
+
+    - name: PHPStan
+      if: matrix.php-versions == '7.2'
```
Comment (atomiix) [inline .github/workflows/php.yml:40]:
```suggestion
      if: matrix.php-versions == '7.3'
```

File: .github/workflows/php.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      run: ./vendor/bin/php-cs-fixer fix --config=.php_cs -v --dry-run
+
+    - name: Lint TWIG
+      run: ./bin/console lint:twig app/Resources
+
+    - name: Lint YAML
+      run: ./bin/console lint:yaml app/config
+
+    - name: Security Check
+      if: matrix.php-versions == '7.2'
```
Comment (atomiix) [inline .github/workflows/php.yml:53]:
```suggestion
      if: matrix.php-versions == '7.3'
```

File: .github/workflows/php.yml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+      run: |
+        curl -L https://github.com/fabpot/local-php-security-checker/releases/download/v1.0.0/local-php-security-checker_1.0.0_linux_386 -o ./security-checker
+        chmod a+x ./security-checker
+        ./security-checker --path=composer.lock
+
+    - name: Upload coverage results to Coveralls
+      if: matrix.php-versions == '7.2'
```
Comment (atomiix) [inline .github/workflows/php.yml:60]:
```suggestion
      if: matrix.php-versions == '7.3'
```

**Review by atomiix — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @Progi1984

**Pull request merged**